infra/ansible/roles/django/tasks/main.yml

```diff
@@ -85,6 +85,7 @@
   become_user: gunicorn
   notify:
     - Migrate Django database
+    - Create database cache
```

Comment:
Similarly, I think that this handler should also be called in "infra/ansible/roles/restore/tasks/main.yml" after restore tasks.

Reply:
Thanks, fixed by aa79456.

Now the handler `Create database cache` is notified each time `Migrate Django database` is notified.